Modules/gcmodule.c

```diff
@@ -269,27 +283,41 @@ Return the list of objects that are directly referred to by 'objs'.
 [clinic start generated code]*/
 
 static PyObject *
-gc_get_referents_impl(PyObject *module, PyObject *args)
-/*[clinic end generated code: output=d47dc02cefd06fe8 input=b3ceab0c34038cbf]*/
+gc_get_referents_impl(PyObject *module, Py_ssize_t nargs,
+                      PyObject *const *args)
+/*[clinic end generated code: output=e459f3e8c0d19311 input=b3ceab0c34038cbf]*/
 {
-    if (PySys_Audit("gc.get_referents", "(O)", args) < 0) {
+    PyObject *varargs = PyTuple_New(nargs);
+
+    if (!varargs) {
+        return NULL;
+    }
+    for (Py_ssize_t i = 0; i < nargs; i++) {
+        PyTuple_SET_ITEM(varargs, i, Py_NewRef(args[i]));
+    }
```

Comment:
Let's use `_PyTuple_FromArray` here; the `gc` module is a core module, and there is no need to avoid internal API, IMO.

Reply:
Ditto for `gc_get_referrers_impl`.